### DISCUSSION TITLE
Расчетное значение ячейки

### DISCUSSION THREAD

**Discussion by tvideo** [totumonline/totum-ru-issues-and-discussions#131] [Как это сделать?] [answered]
Здравствуйте. Подскажите, плз со следующим вопросиком:
есть временная таблица, в ней есть столбец "stoimost", который формируется по вормуле 
= : if(condition: #izmenenie < 0; then: $then1; else: 0)
then1: modul(num: $ResaultMarja)
ResaultMarja: #izmenenie * #vyhod_cena

![image](https://user-images.githubusercontent.com/32614108/168238059-49dc0cee-d86c-46d2-8fe8-8bb93ef50ea5.png)
Почему-то значение не меняется, а появляется значок "Расчетное значение" с нужным расчетным числом.
В кокую сторону нужно думать?
Спасибо

**Comment by erpic-ru** [comment, marked as answer]
Там же пин на строке, т.е. ручное изменение. Надо модифицировать код в котором Pin будет либо сбрасываться, либо запретить ручные изменения по полю.

https://ru.docs.totum.online/functions#fn-clear

**Reply by tvideo** [reply]
Помогло. Спасибо

**Reply by erpic-ru** [reply]
Короче, вы загружаете это значение, а потом оно рассчитывается кодом и не совпадает с тем, что было загружено. Раз уж оно рассчитывается, то самый простой способ - не загружать его

**Reply by ttm-alexeypolunin** [reply]
Я вот придумал что если оно редактируется в поле источнике и здесь, то можно это поле заблочить и открывать на редактирование поле источник по экшену двойного клика функцией `linkToEdit`. Я ни разу так не делал еще — попробую сегодня.
